deployability/modules/testing/tests/test_manager/test_uninstall.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+
+from ..helpers.manager import WazuhManager
+from ..helpers.generic import CheckFiles, HostInformation, GeneralComponentActions
+from ..helpers.constants import WAZUH_ROOT
+
+
+def uninstall_manager_callback(manager_params):
+    WazuhManager.uninstall_manager(manager_params)
+
+
+def perform_action_and_scan_for_manager(manager_params):
+    result = CheckFiles.perform_action_and_scan(manager_params, lambda: uninstall_manager_callback(manager_params))
+    categories = ['/root', '/usr/bin', '/usr/sbin', '/boot']
+    actions = ['added', 'modified', 'removed']
+
+    # Selecting filter
+    os_name = HostInformation.get_os_name_from_inventory(manager_params)
+    if 'debian' in os_name:
+        filter_data= {'/boot': {'added': [], 'removed': [], 'modified': ['grubenv']}, '/usr/bin': {'added': ['unattended-upgrade', 'gapplication', 'add-apt-repository', 'gpg-wks-server', 'pkexec', 'gpgsplit', 'watchgnupg', 'pinentry-curses', 'gpg-zip', 'gsettings', 'gpg-agent', 'gresource', 'gdbus', 'gpg-connect-agent', 'gpgconf', 'gpgparsemail', 'lspgpot', 'pkaction', 'pkttyagent', 'pkmon', 'dirmngr', 'kbxutil', 'migrate-pubring-from-classic-gpg', 'gpgcompose', 'pkcheck', 'gpgsm', 'gio', 'pkcon', 'gpgtar', 'dirmngr-client', 'gpg', 'filebeat', 'gawk', 'curl', 'update-mime-database', 'dh_installxmlcatalogs', 'appstreamcli','lspgpot'], 'removed': [], 'modified': []}, '/root': {'added': ['trustdb.gpg'], 'removed': [], 'modified': []}, '/usr/sbin': {'added': ['update-catalog', 'applygnupgdefaults', 'addgnupghome', 'install-sgmlcatalog', 'update-xmlcatalog'], 'removed': [], 'modified': []}}
+    else:
+        filter_data = {'/boot': {'added': [], 'removed': [], 'modified': ['grubenv']}, '/usr/bin': {'added': ['filebeat'], 'removed': [], 'modified': []}, '/root': {'added': ['trustdb.gpg'], 'removed': [], 'modified': []}, '/usr/sbin': {'added': [], 'removed': [], 'modified': []}}
+
+    # Use of filters
+    for directory, changes in result.items():
+        if directory in filter_data:
+            for change, files in changes.items():
+                if change in filter_data[directory]:
+                    result[directory][change] = [file for file in files if file.split('/')[-1] not in filter_data[directory][change]]
+
+    # Testing the results
+    for category in categories:
+        for action in actions:
+            assert result[category][action] == []
+
```

Comment:
If we move the function from `deployability/modules/testing/tests/test_manager/test_install.py` to a utils file, maybe we can reuse it here with some minor modifications, right?

Reply:
As the action is for install and uninstall, both were moved to manager.py

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5112/commits/dd818b765d551ab2c6431656986ec60cea05225c